src/utils/messages.ts

```
@@ -1,11 +1,17 @@
 import logger from '@/lib/logger';
 import type { FilePart, ModelMessage } from 'ai';
 import {
-  type Attachment as DiscordAttachment,
   type Collection,
+  type Attachment as DiscordAttachment,
   type Message as DiscordMessage,
+  Message,
 } from 'discord.js';
```

ESLint

[error] Unable to resolve path to module 'discord.js'.

(import-x/no-unresolved)